README.md

```diff
@@ -53,6 +53,7 @@ $ firebase functions:config:set coordinator.email_address='EMAIL'
 $ firebase functions:config:set coordinator.timeZoneOffset=NUMBER of HOURs
 #Importing a spreadsheet to the database variables
 $ firebase functions:config:set sqr.spreadsheetId='Google Spreadsheet ID'
+firebase functions:config:set sound_editing.spreadsheet_id="Google Spreadsheet ID"
```

Comment:
There is no “Sound Editing Spreadsheet”. This one should be called “CR Processing Spreadsheet”.

```suggestion
 firebase functions:config:set reporting.content.processing.spreadsheet_id="Google Spreadsheet ID"
```